CRUDRecipeEF.PL/DataSeed.cs

```diff
@@ -0,0 +1,37 @@
+﻿using CRUDRecipeEF.BL.DL.Data;
+using CRUDRecipeEF.BL.DL.Entities;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+
+namespace CRUDRecipeEF.PL
+{
+    public static class DataSeed
+    {
+        public static void Data(this ModelBuilder modelBuilder)
```

Comment:
i found this for you, its a much more common way  of data seeding and it would work for our application even if he is doing it in another. [seed data link](https://www.youtube.com/watch?v=tuZu9M-L5Jc ) but what i would do is the same as him except i would check in the main method if the context has anything before calling the seed method